data-mongodb/src/test/groovy/io/micronaut/data/document/mongodb/MongoManyToManySpec.groovy

```diff
@@ -247,6 +248,7 @@ class CourseRatingCompositeKey {
 
 @EqualsAndHashCode
 @Embeddable
+@Introspected
```

Comment:
Had to add this, without it there was an error:
```
Message: No bean introspection available for type [class io.micronaut.data.document.mongodb.CourseRatingKey]. Ensure the class is annotated with io.micronaut.core.annotation.Introspected
```
although it has `@Embeddable` annotation which includes `@Introspected`.

Reply:
seems like a bug in Groovy again, @dstepanov ?

Reply:
Looks like it is resolved by Core M4.